ui/src/static/core.css

```diff
@@ -11675,3 +11675,12 @@ a.dark-link {
   padding-right: 0px;
 }
 
+@keyframes rotation {
```

Comment:
Ahhh you put that in here. Interesting. Think this would be better in a util in the JavaScript instead?

Reply:
Yeah, I agree that we need to move it to common utils. I did this before the moment when I found proper solution from styled components library docs

Reply:
Okay no worries, do you think we should do that as part of this PR so we don't forget about it?

Reply:
Yeah, it would be nice to keep this stuff together